src/tactic/simps.lean

```diff
@@ -14,9 +14,9 @@ reducing a definition when projections are applied to it.
 
 There are three attributes being defined here
 * `@[simps]` is the attribute for objects of a structure or instances of a class. It will
-  automatically generate simplication lemmas for each projection of the object/instance that contains
-  data. See the doc strings for `simps_attr` and `simps_cfg` for more details and configuration
-  options
+  automatically generate simplication lemmas for each projection of the object/instance that
+  contains data. See the doc strings for `simps_attr` and `simps_cfg` for more details and
+  configuration options.
 * `@[_simps_str]` is automatically added to structures that have been used in `@[simps]` at least
   once. They contain the data of the projections used for this structure by all following
   invocations of `@[simps]`.
```

Comment:
I'm not sure what the referent of `They` is in "They contain the data". Could you add a bit more here?

Reply:
Better now?